docs/src/build.sh

```diff
@@ -39,4 +39,4 @@ DOCS="${DIR}/doc"
 rm -rf "${DOCS}"
 jsdoc ${DIR}/../www/*.js --readme ${DIR}/../src/README.md -d "${DOCS}"
 
-python3 "${DIR}/tsv-dd-to-html.py"
\ No newline at end of file
+# python3 "${DIR}/tsv-dd-to-html.py"
```

Comment:
This change should not be committed as it bypasses the friendly HTML DD.

Reply:
The script call has been restored in build.sh.